module/os/freebsd/zfs/sysctl_os.c

```diff
@@ -144,6 +144,55 @@ extern arc_state_t ARC_l2c_only;
 
 /* arc.c */
 
+int
+param_set_arc_max(SYSCTL_HANDLER_ARGS)
+{
+	uint64_t val;
+	int err;
+
+	val = zfs_arc_max;
+	err = sysctl_handle_long(oidp, &val, 0, req);
+	if (err != 0 || req->newptr == NULL)
+		return (SET_ERROR(err));
+
+	if (val != 0 && (val < 64 << 20 || val <= arc_c_min ||
```

Comment:
This magic `64 << 20` value doesn't feel great, here and in `arc_init()`. Can we use something that explains itself a bit better?

Reply:
Could certainly make it a #define that explains that it means '64 megabytes'

Reply:
More a question of why that value. It appears in multiple places, with no explanation of where it comes from.

Reply:
Using a `#define` would be helpful.  We settled on this value because ZFS would get very unhappy when the maximum ARC size was smaller than the maximum block size (16M).  Since we wanted to leave ourselves some additional margin we opted to use the maximum transaction size, `DMU_MAX_ACCESS`, as the lower bound.

Reply:
I went ahead and added the `#define`.